src/preparser.js

```diff
@@ -12,10 +12,9 @@ const parseEnvVariables = function (input) {
   });
 };
 
-const preparser = function (input) {
-  console.log({input})
-  input = parseEnvVariables(input);
-  return input;
-};
+// const preparser = function (input) {
+//   input = parseEnvVariables(input);
+//   return input;
+// };
 
 module.exports = preparser;
```

Comment:
When we will use it, bash-parser has already taken care of env parameter expansion at this point, so the preparser should be superflous